toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -60,6 +75,10 @@ mknod -m 600 $chroot_builder_folder/dev/console c 5 1
 mknod -m 666 $chroot_builder_folder/dev/null c 1 3
 mknod -m 444 $chroot_builder_folder/dev/urandom c 1 9
 
+# Start with filesystem, then macros, then everything else
```

Comment:
Why do these packages need to be installed first?

Reply:
The filesystem is responsible for laying out the directory structure, and the majority of packages expect it to be installed already. The macros change the behavior of rpm install (by setting non-default paths etc).

Reply:
In this case, I just want to be more defensive about ordering. I'd like to remove the requirement that the manifest file is sorted. I actually removed the parallel install stuff I was doing in the PR which necessitated installing these first, but I dropped that. I can probably just remove these lines for now.